common/src/test/java/com/microsoft/identity/common/internal/broker/AuthUxJsonPayloadObjectTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.microsoft.identity.common.internal.broker
```

Comment:
LICENSE

Reply:
License is there, I think you are on an older commit